libs/ngx-mime/src/lib/core/access-keys-handler-service/access-keys.service.ts

```diff
@@ -40,9 +40,11 @@ export class AccessKeysService {
     this.subscriptions = new Subscription();
     this.subscriptions.add(
       this.iiifManifestService.currentManifest.subscribe(
-        (manifest: Manifest) => {
-          this.isSearchable = this.isManifestSearchable(manifest);
-          this.invert = manifest.viewingDirection === ViewingDirection.RTL;
+        (manifest: Manifest | null) => {
+          this.isSearchable = manifest ? this.isManifestSearchable(manifest) : false;
```

Comment:
This can be moved into the if statement under to avoid checking if manifest exists twice.
invert has a initialized value of false so even if the manifest doesn't exists it should be false.